WebRoot/skins/_base/base/skin.properties

```diff
@@ -54,6 +54,7 @@ LogoImgDir		= @AppContextPath@/skins/_base/logos
 #
 #	LogoImgDir		= @AppContextPath@/skins/@SkinName@/logos
 #	LogoURL			= http://www.yourcompany.com
+#	FaviconDir		= @AppContextPath@/skins/@SkinName@/img/favicon
```

Comment:
we shouldn't change anything in base and base2 directories as those are old base skins and we are using base3 only currently

Reply:
I reverted changes in base and base2 directories.